Source/Blazorise/wwwroot/video.js

```diff
@@ -0,0 +1,66 @@
+﻿import "./vendors/ShakaPlayer";
+import * as utilities from "./utilities.js";
+
+const _instancesInfos = [];
+
+export function initialize(dotnetAdapter, element, elementId, options) {
+    element = utilities.getRequiredElement(element, elementId);
+
+    if (!element)
+        return;
+
+    // Install built-in polyfills to patch browser incompatibilities.
+    shaka.polyfill.installAll();
```

Comment:
I would probably move this up, before the `initialize` method so that it doesn't get called every time a new video is created.